src/test/java/ludogorie_soft/reservations_platform_api/controller/IntegrationTestBase.java

```diff
@@ -0,0 +1,27 @@
+package ludogorie_soft.reservations_platform_api.controller;
+
+import ludogorie_soft.reservations_platform_api.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+public abstract class IntegrationTestBase {
```

Comment:
What is the idea of this TestBase class in long term?
I guess as the project grows for each test class we are going to need different data to be deleted and set up.

Reply:
I removed it!